source/ast.rs

```diff
@@ -55,3 +55,20 @@ pub struct Addition {
     /// The right-hand side of the addition.
     pub b: Term
 }
+
+/// A literal.
```

Comment:
"A literal represents a fixed value in the source code. For example, a string literal would be `'foobar'`."